source/sessions/driver-sessions.rst

```diff
@@ -1269,3 +1267,4 @@ Change log
 :2019-05-15: A ServerSession that is involved in a network error MUST be discarded
 :2019-10-22: Drivers may defer checking if a deployment supports sessions until the first
 operation performed with a session
+:2020-05-26: Simplify logic for determining sessions support, some standalones do support sessions
```

Comment:
Can we be more precise about which standalone servers support sessions?  Was support added in some release after the initial one that included sessions for replica sets?

Reply:
Do you think there are servers which report `logicalSessionsTimeoutMinutes` and don't support sessions? Otherwise, I say we depend completely on its existence for feature detection here, no need to complicate things further

Reply:
That's fine, but it's still confusing to talk about "some standalones".    Maybe just remove everything after the comma?

Reply:
done